applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -159,27 +202,44 @@ static struct bt_audio_stream *lc3_cap_config_cb(struct bt_conn *conn, struct bt
 						 struct bt_codec *codec)
 {
 	int ret;
-	struct bt_audio_stream *stream = &audio_stream;
 
-	if (!stream->conn) {
-		LOG_DBG("ASE Codec Config stream %p", (void *)stream);
-
-		print_codec(codec);
-
-		uint32_t octets_per_sdu = bt_codec_cfg_get_octets_per_frame(codec);
-
-		if (octets_per_sdu > LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MAX)) {
-			LOG_WRN("Too high bitrate");
-			return NULL;
-		} else if (octets_per_sdu < LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MIN)) {
-			LOG_WRN("Too low bitrate");
-			return NULL;
+	for (size_t i = 0; i < ARRAY_SIZE(streams); i++) {
+		struct bt_audio_stream *stream = &streams[i];
+
+		if (!stream->conn) {
+			LOG_DBG("ASE Codec Config stream %p", (void *)stream);
+
+			uint32_t octets_per_sdu = bt_codec_cfg_get_octets_per_frame(codec);
+
+			if (octets_per_sdu > LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MAX)) {
+				LOG_WRN("Too high bitrate");
+				return NULL;
+			} else if (octets_per_sdu <
+				   LE_AUDIO_SDU_SIZE_OCTETS(CONFIG_LC3_BITRATE_MIN)) {
+				LOG_WRN("Too low bitrate");
+				return NULL;
+			}
+
+			if (dir == BT_AUDIO_DIR_SINK) {
+				LOG_DBG("BT_AUDIO_DIR_SINK");
+				print_codec(codec);
+				ret = ctrl_events_le_audio_event_send(LE_AUDIO_EVT_CONFIG_RECEIVED);
+				ERR_CHK(ret);
+			}
+#if CONFIG_STREAM_BIDIRECTIONAL
+			else if (dir == BT_AUDIO_DIR_SOURCE) {
+				LOG_DBG("BT_AUDIO_DIR_SOURCE");
+				print_codec(codec);
+				source_streams[configured_source_stream_count++].stream = stream;
```

Comment:
It is abit strange to "support multiple source streams" here, when index '0' is hardcoded in le_audio_send()

Reply:
Agree, so we need to discuss if we want this scalable or restricted